content/blog/2024-06-19-ethereum-classic-vs-central-bank-digital-currencies-CBDC-s/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Ethereum Classic vs Central Bank Digital Currencies (CBDCs)"
+date: 2024-06-19
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["philosophy"]
+linkImage: ./banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/IDr_mPliX70" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+![](./banner.png)
+
+As central banks all over the world race to launch their Central Bank Digital Currencies (CBDCs) to please the global elites such as the [World Economic Forum](https://www.weforum.org/publications/modernizing-financial-markets-with-wcbdc/), we as citizens have no other recourse but to learn and understand what these new technologies will entail to our life, liberty, and property.
+
+To explain what are CBDCs, a good way of going about it is to compare them with Ethereum Classic (ETC).
+
+ETC is the most secure and advanced blockchain technology in the world and CBDCs seek to imitate its design but to add to them their central control and planning devices.
+
```

Comment:
seems like CBDC is trying to copy ETH not ETC

Reply:
Well, ETC is the original EVM chain so I made this sarcastic statement because imitating ETH is to imitate ETC. But a few paragraphs after this I clarify or reveal my sarcasm by stating that ETH and ETC are EVM cahins and that CBDCs are imitating the EVM model!